tests/functional/test_api.py

```diff
@@ -6,73 +6,46 @@
 from ghmirror.app import APP
 from ghmirror.data_structures.monostate import UsersCache
 
+class MockResponse:
```

Comment:
two blank lines between the imports and the class definition (PEP8 - we should start checking the tests :/)

Reply:
my bad. I was relying on the linter to catch these.